NanoGardener/python/modules/BWEwkSingletReweighter.py

```diff
@@ -274,11 +274,11 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
 
         self.out = wrappedOutputTree
         self.branchnames = []
-        #self.branches = ["_I", "_B", "_I_Honly", "_I_Bonly", "_I_HB", "_H"] # Don't really need all of them
-        if self.finalState == "LNuQQ":
-          self.branches = ["_I", "_B"]
-        else:
-          self.branches = ["_I"]
+        self.branches = ["_I", "_B", "_I_Honly", "_I_Bonly", "_I_HB", "_H"] # Don't really need all of them
+        #if self.finalState == "LNuQQ":
+        #  self.branches = ["_I", "_B"]
+        #else:
+        #  self.branches = ["_I"]
```

Comment:
I made the experience that adding too many branches will cause problems. If the output filesize is too big, further postprocessing (scale uncertainties) will fail without doing really convoluted things (See comment in our Mattermost "Postprocessing" channel from May 13, 2022).
If you need all of these branches, I'd suggest considering less width scenarios, in line 16. Ultimately, for the di-leptonic analysis, I only used the 0.00, 0.01, 0.02, 0.05, 0.10 relative width scenarios.

Reply:
Thanks! I am moving the _B and _H weights outside of the loop over widths, because these do not depend on the width you reweight to. Hopefully this will reduce the number of branches. For the moment I have put them all back in for debugging purposes. I may remove the debugging ones if not needed afterwards.